modules/setting/repository.go

```diff
@@ -199,6 +200,7 @@ var (
 			DefaultMergeMessageOfficialApproversOnly bool
 			PopulateSquashCommentWithCommitMessages  bool
 			AddCoCommitterTrailers                   bool
+			RetargetChildsOnClose                    bool
```

Comment:
```suggestion
			RetargetChildrenOnClose                    bool
```
Also when you add configuration options, please also add them to `custom/conf/app.example.ini` and `docs/content/doc/advanced/config-cheat-sheet.en-us.md`.